<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="WIN-81NG0PGA72A" name="TestNGPrograms.SoftAssertionZeroWebApp" tests="7" failures="1" timestamp="24 May 2020 15:17:58 GMT" time="6.485" errors="0">
  <testcase name="Browser" time="6.288" classname="TestNGPrograms.SoftAssertionZeroWebApp"/>
  <testcase name="assertEquals" time="0.139" classname="TestNGPrograms.SoftAssertionZeroWebApp">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [ONLINE BANKINGG] but found [ONLINE BANKING]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [ONLINE BANKINGG] but found [ONLINE BANKING]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at TestNGPrograms.SoftAssertionZeroWebApp.assertEquals(SoftAssertionZeroWebApp.java:31)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- assertEquals -->
  <testcase name="assertNotEquals" time="0.001" classname="TestNGPrograms.SoftAssertionZeroWebApp"/>
  <testcase name="assertTrue" time="0.038" classname="TestNGPrograms.SoftAssertionZeroWebApp"/>
  <testcase name="assertFalse" time="0.001" classname="TestNGPrograms.SoftAssertionZeroWebApp"/>
  <testcase name="assertNotNull" time="0.017" classname="TestNGPrograms.SoftAssertionZeroWebApp"/>
  <testcase name="assertNull" time="0.001" classname="TestNGPrograms.SoftAssertionZeroWebApp"/>
</testsuite> <!-- TestNGPrograms.SoftAssertionZeroWebApp -->
